doc/cprover-manual/memory-predicates.md

```diff
@@ -0,0 +1,54 @@
+# Memory Predicates
+
+
+### Definition
+`__CPROVER_is_fresh` specifies freshly allocated objects.
```

Comment:
"freshly" is not defined.

It's worth mentioning that each such allocation must be distinct from _all_ other allocations.

Reply:
Update it.